core/server/master/src/main/java/alluxio/master/service/rpc/RpcServerStandbyGrpcService.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.service.rpc;
+
+import alluxio.master.Master;
+import alluxio.master.MasterProcess;
+import alluxio.master.MasterRegistry;
+
+import com.google.common.base.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetSocketAddress;
+
+/**
+ * Created by {@link RpcServerService.Factory}.
+ * Manages the behavior of the master's rpc service. The grpc server is always on.
+ * When the promotion/demotion happens, the rpc service will be stopped and restarted.
+ * The new started grpc service will serve gRPC endpoints based on the node state (PRIMARY/STANDBY).
+ * No rejecting server is deployed.
+ */
+public class RpcServerStandbyGrpcService extends RpcServerService {
```

Comment:
please review this naming...
basically this is a rpc server service where rpc servers are enabled on all masters.

Reply:
i'm okay with this name